http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -395,6 +395,9 @@ object Boot extends IOApp {
       credential <- credentialResource(pathToCredentialJson)
       scopedCredential = credential.createScoped(Seq(ComputeScopes.COMPUTE).asJava)
       kubernetesScopedCredential = credential.createScoped(Seq(ContainerScopes.CLOUD_PLATFORM).asJava)
+      cloudPlatformScopedCredential = credential.createScoped(
+        "https://www.googleapis.com/auth/cloud-platform"
+      ) // See https://developers.google.com/identity/protocols/oauth2/scopes
```

Comment:
Assuming `cloud-platform` is the minimum required scope based on

![image](https://user-images.githubusercontent.com/5438223/143248081-a8aba0a3-18d2-4fcc-9d8a-d687fd881eb2.png)